Firebase/Auth/Source/FIRAuthDefaultUIDelegate.m

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Please add this file to the macOS exclusion in Auth's podspec so it is not compiled for macOS.

Reply:
Done